build.gradle.kts

```diff
@@ -1,46 +1,44 @@
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.gradle.api.tasks.testing.logging.TestLogEvent
-import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
 plugins {
-    kotlin("jvm") version "1.8.21"
-}
-
-dependencies {
-    testImplementation("org.junit.jupiter:junit-jupiter-engine:5.4.0")
-    testImplementation("com.google.guava:guava:26.0-jre")
+    buildsrc.conventions.lang.`kotlin-multiplatform-jvm`
+    buildsrc.conventions.lang.`kotlin-multiplatform-js`
+    //buildsrc.conventions.lang.`kotlin-multiplatform-native`
```

Comment:
It's commented out, should it stay here?

Reply:
It'll be enabled eventually, but I didn't want to add _loads_ of Kotlin/Native targets right now because it will make development slower - just having two targets is enough to make sure that commonMain doesn't contain any JVM code.